stroyprombeton/tests/tests_selenium.py

```diff
@@ -49,17 +49,17 @@ def buy_on_product_page(
         self, *, option: stb_models.Option=None, quantity=None, waiting_time=1
     ):
         option = option or stb_models.Option.objects.first()
-        product_page = urljoin(self.live_server_url, option.url)
-        self.browser.get(product_page)
+        url = urljoin(self.live_server_url, option.url)
```

Comment:
It would be nice to create a function for this
```python
        url = urljoin(self.live_server_url, option.url)
        self.browser.get(url)
```